plugins/flytekit-dbt/setup.py

```diff
@@ -4,10 +4,7 @@
 
 microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
 
-plugin_requires = [
-    "flytekit>=1.3.0b2",
-    "dbt-core<1.8.0",
-]
+plugin_requires = ["flytekit>=1.3.0b2", "dbt-core>=1.6.0,<1.8.0", "networkx>=2.5"]
```

Comment:
can you talk about more why we need`networkx`?

Reply:
Thank you for review ! And I added the explanation .